flips/20220817-add-fields-to-existing-definitions.md

```diff
@@ -0,0 +1,80 @@
+# Allow new fields in deployed Resources and Structs
+
+| Status        | Proposed                                                  |
+|---------------|:--------------------------------------------------------- |
+| **FLIP #**    | [NNN](https://github.com/onflow/flow/pull/NNN)            |
+| **Author(s)** | Deniz Mert Edincik (deniz@edincik.com)                    |
+|               | Austin Kline (austin@flowty.io)                           | 
+| **Sponsor**   |                                                           |
+| **Updated**   | 2022-08-17                                                |
+
+## Objective
+This proposed change will allow existing structs and resources to add new fields to them by
```

Comment:
Would it be beneficial to allow updating contracts with new methods as well? Or just fields?

Reply:
Right, functions can already be added